doc/api/fs.md

```diff
@@ -1747,6 +1747,10 @@ fs.copyFileSync('source.txt', 'destination.txt', COPYFILE_EXCL);
 <!-- YAML
 added: v0.1.31
 changes:
+  - version:
+     - v15.0.2
```

Comment:
```suggestion
     - REPLACEME
```

Please use this placeholder instead of `v15.0.2` -- it will be replaced by the correct version on release.